### PR TITLE
fixes usability issues with calling experience

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**
Extracted audio call to its own component.
Blocks double-clicking.

image

### CODE REVIEW

File: src/components/Inbox/InboxHeader.tsx [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-import ConnectionInfo from "./ConnectionInfo";
-import * as Sentry from "@sentry/react";
 import IconButton from "../Common/Buttons/IconButton";
+import ConnectionInfo from "./ConnectionInfo";
+import { TConnectionListItem } from "./ConnectionListSidebar/ConnectionListItem";
 
 interface Props {
   connection: TConnectionListItem;
   onCall: () => Promise<void>;
+  isLoading?: boolean;
```
Comment (cainlevy) [inline src/components/Inbox/InboxHeader.tsx:14]:
```suggestion
  callIsLoading?: boolean;
```

File: src/components/Inbox/VoiceCall/VoiceCallInterface.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        onTerminate();
+        showToast("callStatus", t("call:callStatus.ended"), "info");
+        break;
+      default:
+        break;
+    }
+  }, [onTerminate, t, status]);
+  const peer = peers[0];
+
+  console.log("vc", vc);
```
Comment (cainlevy) [inline src/components/Inbox/VoiceCall/VoiceCallInterface.tsx:82]: ✂️

**Review by cainlevy — APPROVED** [review]
i continue to be surprised by how much manual cache maintenance is involved to maintain the timeline

File: src/components/Inbox/VoiceCall/VoiceCallInterface.tsx (hunk 2 above)
Comment (gabrielsaruhashi) [reply src/components/Inbox/VoiceCall/VoiceCallInterface.tsx:82]: ahhh forgot to push the amendment fixing this

**Pull request merged**